_data/banking.yml

```
@@ -495,6 +611,8 @@ websites:
       - sms
       - proprietary
     doc: https://www.desjardins.com/securite/validation-2-etapes/
+    regions:
+      - ca
```

Florida is covered as well - does that deserve a "us" entry?
```suggestion
      - ca
      - us
```